icu4j/maven-migration/toMaven.sh

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env bash
+
+# Move the sources from main/classes and main/tests into a maven structure
+# - `main/classes/foo/src/java` go to `foo/src/main`, then split into `java` and `resources`
+# - `main/tests/foo/src/java` go to `foo/src/test`, then split into `java` and `resources`
+# - `main/tests/foo/src/META-INF` goes to ... `resources`
+#
+# From this:
+# classes/
+#   <module>/
+#     src/
+# test/
+#   <module>/
+#     src/
+#
+# To this (standard maven structure):
+# <module>/
+#   main/
+#     src/
+#       java/
+#       resources/
+#   test/
```

Comment:
Current folder is `main/tests`. Maven wants `src/test`.
I've updated the plan document shared today.
I've checked this file again, I think it's fine.
It is hard to see it, as because old used `tests`, new uses `test`, so I have to be very careful.

But the words that can happen is an incorrect comment in a temporary script.
The code part is fine, because it was tested many times.